bot/handlers.go

```diff
@@ -22,7 +22,6 @@ func (b *bot) MsgReceived(msg msg.Message) {
 
 	// msg := b.buildMessage(client, inMsg)
 	// do need to look up user and fix it
-
```

Comment:
I think this used to look for messages from the bot and trigger selfSaid instead. Or possibly the irc stuff did that. It's possible this self-aware bug has been around for longer than we thought.